CMakeLists.txt

```diff
@@ -183,48 +183,48 @@ set(GOST_ENGINE_SOURCE_FILES
         )
 
 add_executable(test_digest test_digest.c)
-target_link_libraries(test_digest gost_engine gost_core ${OPENSSL_CRYPTO_LIBRARY})
+target_link_libraries(test_digest OpenSSL::Crypto)
 add_test(NAME digest
 	COMMAND test_digest)
 
 add_executable(test_curves test_curves.c)
-target_link_libraries(test_curves gost_engine gost_core ${OPENSSL_CRYPTO_LIBRARY})
+target_link_libraries(test_curves gost_engine gost_core)
```

Comment:
You can drop `gost_core` here

Reply:
Done

Reply:
I think OpenSSL::Crypto should be added here. test_curves.c calls OpenSSL functions and I can't understand why u removed OpenSSL::Crypto in ur commit...

Reply:
Dropped gost_core and added OpenSSL::Crypto